lib/python/operations.cpp

```diff
@@ -80,6 +81,83 @@ void bind_midpoints(py::module &m) {
   });
 }
 
+void bind_experimental(py::module &m) {
+  m.def("experimental_transform", [](py::object const &kernel,
+                                     const Variable &a) {
+    auto fptr_address = kernel.attr("address").cast<intptr_t>();
+    auto fptr = reinterpret_cast<double (*)(double)>(fptr_address);
```

Comment:
If we make this public, we should add a mechanism for determining the actual argument types and at least checking if they are all `double`. There should be something in `kernel.attr`...

Reply:
Well, it is not really public, it is in `_scipp.core`.